go/client/cmd_chat_send.go

```diff
@@ -138,14 +114,15 @@ func (c *cmdChatSend) ParseArgv(ctx *cli.Context) (err error) {
 	c.setTopicName = ctx.String("set-topic-name")
 	c.setHeadline = ctx.String("set-headline")
 	c.clearHeadline = ctx.Bool("clear-headline")
-	c.useStdin = ctx.Bool("stdin")
 	c.nonBlock = ctx.Bool("nonblock")
 
 	var tlfName string
-	// Get the TLF name from the first position arg
-	if len(ctx.Args()) >= 1 {
-		tlfName = ctx.Args().Get(0)
+	if len(ctx.Args()) < 1 {
+		return errors.New("TLF name empty. What conversation do you want to send to?")
```

Comment:
I don't think any users know what a TLF is.  `"Conversation participants empty."` or something like that.